Prace_domowe/Praca_domowa7/Gruszkowska_Klaudia/PD 7.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>Super, że odniesiono się do sposobu w jaki wyliczane jest precision i jak to wpływa na wynik</p><p> </p><p>Generalnie dobrze wykonana praca domowa. Widać duży nakład pracy koncepcyjnej! Zastosowano wiele ciekawych analiz!</p><p>Jednak zdarzył się mały błąd ze zdefiniowaniem liczby klastrów.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/348/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>